### DISCUSSION TITLE
retrieve some data of GitHub users

### DISCUSSION THREAD

**Discussion by saharCoder** [community/community#29417] [Apps, API and Webhooks]
I am working on a project and I need to retrieve some data of GitHub users, I want to retrieve the data of Saudi users whose location is  “Saudi Arabia” on their pages, get the number of stars for each Saudi user, and the number of commits and collaborators on "open source ai packages",
I tried to do it but I did get a limited number of Saudi users back, not all of them. 
I hope you can help me, thanks

**Comment by Wep-wep** [comment]
Yo quiero que alguien me ayude necesito 5 millones dlls para invertir estoy dispuesto a suicidarme quizás Haci lo llegué a lograr. Volviendo a su boletin, me encantaría ayudarle pero yo soy mexicano.... De cualquier manera estoy para servirle My friend
